solutions/csharp/01-init/code/src/Program.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.IO;
+
+static bool MatchPattern(string inputLine, string pattern) {
+    if (pattern.Length == 1) {
+        return inputLine.Contains(pattern);
+    } else {
+        throw new ArgumentException($"Unhandled pattern: {pattern}");
+    }
+}
+
+if (args[0] != "-E") {
+    Console.WriteLine("Expected first argument to be '-E'");
+    Environment.Exit(2);
+}
+string pattern = args[1];
```

Comment:
@ryan-gang can you confirm how we reviewed formatting here? How do we know that this is C# code that will look "well-formatted" to a senior dev?